nixos/modules/system/activation/switch-to-configuration.pl

```diff
@@ -997,14 +1027,13 @@ sub filter_units {
         }
     }
 
-    # Reloading containers & dbus.service in the same transaction causes
-    # the system to stall for about 1 minute.
```

Comment:
Note: this comment still seems relevant, but I'll just re-add this after merging.